docs/user/glossary.rst

```diff
@@ -39,36 +44,97 @@ so that you have a reference for how we're using them.
       and rules for publication of documentation can be :doc:`automated </automation-rules>`.
       Similar to :term:`Docs as Code`.
 
+   pinning
+      *Pinning* software requirements is the most important technique to make a project :term:`reproducible`.
+      To *pin* a requirement means to specify which version that should be used.
+
+      When documentation builds,
+      software dependencies are installed in their latest versions permitted by the version specification.
+      Since software packages are frequently released,
+      we are usually trying to avoid incompatibilities in a new release from suddenly breaking a documentation build.
+
+      Examples of Python dependencies::
+
+          # Exact pinning: Only allow Sphinx 5.3.0
+          sphinx==5.3.0
+
+          # Loose pinning: Lower and upper bounds result in the latest 5.3.x release
+          sphinx>=5.3,<5.4
+
+          # Very loose pinning: Lower and upper bounds result in the latest 5.x release
+          sphinx>=5,<6
+
+      Read the Docs recommends using **exact pinning**.
+
+      See: :doc:`/guides/reproducible-builds`.
+
    pre-defined build jobs
       Commands executed by Read the Docs when performing the build process.
       They cannot be overwritten by the user.
 
-   profile page
-      Page where you can see the projects of a certain user.
-
    project home
       Page where you can access all the features of Read the Docs,
       from having an overview to browsing the latest builds or administering your project.
 
    project page
       Another name for :term:`project home`.
 
+   reproducible
+      Being "reproducible" is an important positive quality goal of documentation.
+
+      A documentation project is said to be *reproducible* when its sources build correctly on Read the Docs over a periode of many years.
```

Comment:
I'm not sure this definition is the correct one. Being reproducible means to be able to re-build an old version at any time; but here we are saying that it means to be able to build the docs over many years -- which is confusing, in my opinion.

Reply:
Agreed!!